src/backend/aspen/api/conftest.py

```diff
@@ -92,12 +92,7 @@ async def async_sqlalchemy_interface(
     # Insert any core level data we're going to need for tests.
     session = test_db_interface.make_session()
     roles = [Role(name="admin"), Role(name="viewer"), Role(name="member")]
-    pathogens = [
```

Comment:
This is where we *were* adding pathogens to the db. Tests are now responsible for handling their own pathogen data